index.html

```diff
@@ -15,29 +15,32 @@
   <div class="page1Container">
     <header class="header">
       <a href="">
-        <p class="welcome"> <strong>Welcome</strong></p>
+        <p class="welcome collapse"> <strong>Welcome</strong></p>
       </a>
       <a href="" class="desktop_menu">Portfolio</a>
       <a href="#about" class="desktop_menu">About</a>
       <a href="#contact" class="desktop_menu">Contact</a>
-      <div class="vector_container">
-        <a href="">
+      <a href="#" id="burgerMenu">
+        <div class="vector_container">
           <svg class="vectSvg vect1">
-            <rect class="vector" height="2" width="16" />
+            <line x1="0" y1="0" x2="18" y2="0" class="vectLine collapse" />
+            <line x1="0" y1="0" x2="18" y2="12" class="vectCross collapse hide" />
+            <line x1="0" y1="12" x2="18" y2="0" class="vectCross collapse hide" />
```

Comment:
- [ ] please make sure to use the right close button from Figma design in order to match the design I encourage you to do that 
- Figma
![image](https://user-images.githubusercontent.com/61790587/173459054-9231bdb6-f077-41e2-84f1-61ef1ec04641.png)
- Yours
![image](https://user-images.githubusercontent.com/61790587/173459073-85bf5e96-ee43-47cb-afe3-235b5e89b125.png)